src/auth/index.js

```diff
@@ -27,6 +27,9 @@ router.post('/register', requireFields('email', 'password', 'firstName'), (req,
     .catch(next);
 });
 
+router.get('/google', social.google.authenticate);
+router.get('/google/callback', social.google.callback, (req, res) => res.redirect('/'));
```

Comment:
possible feature - redirect on page from `req.query.next` - page where user was, when clicked on `login`

Reply:
that would be nice, agree